<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SeverWindow</class>
 <widget class="QMainWindow" name="SeverWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>369</width>
    <height>940</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="widget_center" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>371</width>
      <height>131</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background:rgb(85, 255, 255);</string>
    </property>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>131</width>
       <height>111</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>楷体</family>
       <pointsize>20</pointsize>
      </font>
     </property>
     <property name="text">
      <string>网上预订</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pb_opensever">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>0</y>
       <width>94</width>
       <height>121</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>楷体</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true"> QPushButton {
color: #960000;
border: 2px solid #000000;
font: 12pt &quot;楷体&quot;;
border-radius: 17px;
padding: 5px;
background: qradialgradient(cx: 0.3, cy: -0.4,
fx: 0.3, fy: -0.4,
radius: 1.63, stop: 0 #ffff00, stop: 1 #ffff7f);
min-width: 80px;
}

QPushButton:hover {
background: qradialgradient(cx: 0.3, cy: -0.4,
fx: 0.3, fy: -0.4,
radius: 1.63, stop: 0 #ffff00, stop: 1 #55ffff);
}

 QPushButton:pressed {
background: qradialgradient(cx: 0.4, cy: -0.1,
fx: 0.4, fy: -0.1,
radius: 1.63, stop: 0 #ffff00, stop: 1 #55ff00);
}</string>
     </property>
     <property name="text">
      <string>开启服务</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>10</y>
       <width>111</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>楷体</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>监听端口：</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_port">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>50</y>
       <width>111</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>楷体</family>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="styleSheet">
      <string notr="true">background:rgb(255, 255, 255);</string>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>160</y>
      <width>201</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>楷体</family>
      <pointsize>22</pointsize>
     </font>
    </property>
    <property name="text">
     <string>请求列表</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>520</y>
      <width>161</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>楷体</family>
      <pointsize>22</pointsize>
     </font>
    </property>
    <property name="text">
     <string>预定列表</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="tableWidget_request">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>210</y>
      <width>351</width>
      <height>261</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>黑体</family>
      <pointsize>12</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QTableWidget" name="tableWidget_booking">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>610</y>
      <width>351</width>
      <height>281</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>黑体</family>
      <pointsize>12</pointsize>
     </font>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
